sdk/attestation/attestation/src/index.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 /*
```

Comment:
👀

Reply:
Would you rather this be `/* */` comments instead of `//` comments? 

Github is marking your feedback as "outdated", if that matters.

Reply:
Nope, ignore my comment. You provided context earlier and I just saw it.

Reply:
To be fair, I saw your version of the change in your PR and I said "Oh, yeah - I should fix that :)"